Array/readme.md

```diff
@@ -26,6 +26,7 @@ To create an array, you need to known the length (or size) of the array in advan
 
  * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
  * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
+ * K Largest Element  ----> [C++](/Code/C++/K_largest_element.cpp)
  * Job Sequencing Problem ----> [C++](/Code/C++/job_sequencing_problem.cpp)
```

Comment:
And do sort this list in lexicographical order

Reply:
@Mugdha-Hazra  This is left.

Reply:
done @rish-singhal sir.

Reply:
Find duplicate in array ~ should be at the top